fabric-data-attachment-api-v1/src/main/java/net/fabricmc/fabric/impl/attachment/AttachmentEntrypoint.java

```diff
@@ -0,0 +1,13 @@
+package net.fabricmc.fabric.impl.attachment;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.entity.event.v1.ServerPlayerEvents;
+
+public class AttachmentEntrypoint implements ModInitializer {
+	@Override
+	public void onInitialize() {
+		ServerPlayerEvents.COPY_FROM.register((oldPlayer, newPlayer, alive) ->
+				AttachmentTargetImpl.copyOnRespawn((AttachmentTargetImpl) oldPlayer, (AttachmentTargetImpl) newPlayer, alive)
```

Comment:
Maybe then iface inject `AttachmentTargetImpl` instead of `AttachmentTarget`?

Reply:
No, as its name suggests, `AttachmentTargetImpl` is implementation-only and shouldn't be exposed to users with interface injection. This cast is fine, as this call is made in impl as well.